doc/nrf/ug_zigbee_commissioning.rst

```diff
@@ -0,0 +1,203 @@
+A node can have one or more of these bits specified.
+
+The following figure shows a generic scenario where where all modes are used, together with their corresponding ``bdbCommissioningMode`` mask bits.
+
+.. figure:: /images/zigbee_commissioning_overview.svg
+   :alt: Simplified top-level Zigbee commissioning procedure
+
+   Generic top-level Zigbee commissioning procedure
+
+This top-level procedure defines the intended order of execution of Zigbee commissioning modes.
+The commissioning procedure is common for all device roles, but some modes might not be executed for a given role.
+For example, Network Formation is not executed for End Devices.
+
+See the following sections for details about each mode.
+
+.. _zigbee_commissioning_modes_touchlink:
+
+Touchlink
+=========
+
+.. note::
+      The Touchlink mode is optional in Zigbee and is not supported in the |NCS|.
+
+The Touchlink mode allows commissioning of devices in close proximity to each other, which means that the key criterium for this mode is the signal strength.
+It involes an initiator device and a target device, where the initiator is a member of an existing Zigbee network or it can form a new network.
+Once the network is created, the initiator looks for the target on four channels by sending InterPAN frames.
+When the target is found, the initiator can create a local binding with a remote endpoint or request adding the remote endpoint to a group.
+The target first identifies itself with the initiator and then receives requests from it and responds to them.
+
+Touchlink only supports `Distributed security network`_ and it is provided as a ZCL cluster, which needs to be implemented on both the initiator and the target devices.
+
+.. _zigbee_commissioning_modes_ns:
+
+Network Steering
+================
+
+The Network Steering mode is used for finding open networks that a device can attempt to join or open a network for a specified period of time.
+
+.. figure:: /images/zigbee_commissioning_steering.svg
+   :alt: Zigbee Network Steering procedure
+
+   Zigbee Network Steering procedure
+
+How the node acts in this mode depends on whether the node is in a Zigbee network or not:
+
+* If the node is already a member of a Zigbee network, it opens the network and then closes it after the specified period of time (180 seconds by default).
+  The opening of the network is triggered when any of the Router or Coordinator nodes broadcasts the permit join request.
+* If the node is not a member of a Zigbee network, it scans channels for an open network to associate with.
+  Once it finds a suitable network, it attempts to join it and get the credentials.
+  If the node finds more than one available network, it will attempt to join them individually.
+
+If the node does not find a network to join and it has either the Coordinator or the Router role, it can attempt to form a new network.
+If the node has the End Device role, its commissioning procedure stops here.
```

Comment:
Technically, Finding and Binding is the last possible step in the commissioning procedure

Reply:
Also for End Devices? Hmmm, @tomchy , should we change this? I remember you said it's ok.